statina/API/v2/endpoints/sample.py

```diff
@@ -0,0 +1,261 @@
+import io
+from datetime import datetime
+from pathlib import Path
+from typing import Dict, List, Literal, Optional
+
+from fastapi import APIRouter, Depends, Form, Query, Security
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import FileResponse, JSONResponse
+
+from statina.adapter import StatinaAdapter
+from statina.API.v2.endpoints.user import get_current_active_user
+from statina.config import get_nipt_adapter
+from statina.crud import update
+from statina.crud.find import find
+from statina.crud.find.plots.zscore_plot_data import (
+    get_abn_for_samp_tris_plot,
+    get_normal_for_samp_tris_plot,
+    get_sample_for_samp_tris_plot,
+)
+from statina.models.database import Batch, DataBaseSample, User
+from statina.models.server.plots.ncv import Zscore131821, ZscoreSamples
+from statina.models.server.sample import Sample
+from statina.parse.batch import validate_file_path
+
+router = APIRouter(prefix="/v2")
+
+
+@router.get("/samples/", response_model=List[Sample])
+def samples(
+    page_size: Optional[int] = Query(5),
+    page_num: Optional[int] = Query(0),
+    current_user: User = Security(get_current_active_user, scopes=["R"]),
+    adapter: StatinaAdapter = Depends(get_nipt_adapter),
+):
+    """Get samples"""
+    samples: List[DataBaseSample] = find.samples(
```

Comment:
You are not returning Sample objects here, but DataBaseSample. Se: https://github.com/Clinical-Genomics/statina/blob/226ad7a63fb234581f0958bfdde5271c68e39839/statina/API/internal/api/api_v1/endpoints/sample.py#L36